### PR TITLE
move the plug-in into Git menu

### PR DESCRIPTION
**Pull request by liyiran**
not quite sure if it is a correct way to do it but it is working locally. Like this plug-in very much.

### CODE REVIEW

**Comment by liyiran** [comment]
for https://github.com/PavlikPolivka/gitcleaner/issues/13

File: src/main/resources/META-INF/plugin.xml
Comment (wimsymons) [inline src/main/resources/META-INF/plugin.xml:33]: Probably you only need to add this line and keep the rest to make it backward/forward compatible.

In the meantime you can use Find Action (https://www.jetbrains.com/help/idea/searching-everywhere.html#find_action) and search for "Delete old branches" to run it.

Comment (liyiran) [reply src/main/resources/META-INF/plugin.xml:33]: updated. Thank you for your suggestion.

Comment (wimsymons) [reply src/main/resources/META-INF/plugin.xml:33]: @PavlikPolivka could you please make a new release? This would make it appear again in the menu of the latest IntelliJ IDEA 2020.3.

**Review by wimsymons — APPROVED** [review]

File: src/main/resources/META-INF/plugin.xml
Comment (primozi) [inline src/main/resources/META-INF/plugin.xml]: I built it and tried to install it on Android Studio 4.1.1 (latest ATW), but I get error that it is incompatible (`Plugin 'Git Branch Cleaner' is incompatible with this installation`).

This change fixes the issue and also keeps IDEA 2020.3 working:
```suggestion
    <idea-version since-build="201"/>
```

If you want it to be compatible with older versions as well (maybe other IDEA based IDEs are even older), this number needs to be even smaller (or even left unchanged).

Comment (liyiran) [reply src/main/resources/META-INF/plugin.xml]: thank you very much. My bad, I did not consider Android Studio.

File: src/main/resources/META-INF/plugin.xml
Comment (primozi) [reply src/main/resources/META-INF/plugin.xml:33]: Just an observation:
because older versions of IDEs do not find `group-id` `Git.MainMenu`, action is placed under menu `Other`.
And because action is added to 'old' groups as well, it appears on both places :)

Comment (liyiran) [reply src/main/resources/META-INF/plugin.xml:33]: Interesting. do you have any better solution? I am just a web-developer and do not have too much experience about plug-in. This plug-in is really useful to me so I hope it will be released soon.

Comment (primozi) [reply src/main/resources/META-INF/plugin.xml:33]: AFAIK it can only be done by uploading multiple builds that correspond to different versions. I'm not developing plugins either :)

That being said... I might correct myself about `since-build` parameter. I read a bit about plugins and if I understand correctly, versions below that `since-build` use older version of plugin. 
Maybe @PavlikPolivka knows more about that? Updated version does cause error to be logged in IDE for not finding the correct `group-id`. Which is not really an issue, but someone will report it here. I'm sure of it.

Comment (liyiran) [reply src/main/resources/META-INF/plugin.xml:33]: I agree with the idea. The older Intellij should use older version of the plug in. So it might be good to have original `since-build` for the latest version. But it is up to @PavlikPolivka to make the decision.

Comment (PavlikPolivka) [reply src/main/resources/META-INF/plugin.xml:33]: Sorry, Iwas completely out for a month.

Yes udating since would prevent the new version for old versions. I am fine with doing it. 

Will merge this after new year.

**Comment by FelixGraf** [comment]
@PavlikPolivka was this change already deployed to the JetBrains Marketplace? I just stumbled across this extension and think it could be really helpful! Unfortunately the menu item is not (yet?) included in the **Git** menu on macOS PhpStorm 2020.3.
Thanks for this great extension.

**Comment by ferdiaob** [comment]
@FelixGraf, right click anywhere on the code, Git > Delete Old Branches.

I can't find it in the menu items at the top of the window, but so long as it's accessable it's good enough for me!

**Comment by wimsymons** [comment]
Indeed, 1.1.5 still not released 😢 https://plugins.jetbrains.com/plugin/10059-git-branch-cleaner/versions
Please @PavlikPolivka make it happen! 🙏

**Comment by miclefebvre** [comment]
Please, I need the new version too!!!

**Comment by miclefebvre** [comment]
> @FelixGraf, right click anywhere on the code, Git > Delete Old Branches.
> 
> I can't find it in the menu items at the top of the window, but so long as it's accessable it's good enough for me!

Just saw your comment to access it somewhere else. Thanks you're my savior.

**Comment by ferdiaob** [comment]
> > @FelixGraf, right click anywhere on the code, Git > Delete Old Branches.
> > I can't find it in the menu items at the top of the window, but so long as it's accessable it's good enough for me!
> 
> Just saw your comment to access it somewhere else. Thanks you're my savior.

No worries pal. I couldn't believe how lost I was without this handy extension myself :)

**Pull request merged**
